### ISSUE TITLE
win10 pro failed install/uninstall

### ISSUE THREAD

**Issue by jocelynjyl** [Alexpux/MSYS2-packages#1370]
Followed the instructions here https://www.msys2.org/ using msys2-x86_64-20180531.exe to install on 64-bit Windows 10 Pro.

`pacman -Syu` succeeds. Then I restart the PC. 

However, after running `pacman -Su`, bash disappears from usr/bin and re-opening msys2 gives "Failed to run '/usr/bin/bash': No such file or directory". Also, Msys2 fails to uninstall cleanly; the entry is left behind in Apps & Features and there's no way to remove it.

**Comment by StarWolf3000** [comment]
What kind of anti-virus software do you have installed?

**Comment by jocelynjyl** [comment]
Avast Free 
Version 18.1.2326 (build 18.1.3800.0)

**Comment by StarWolf3000** [comment]
Is the MSYS2 installation folder on both Exclude lists?

**Comment by jocelynjyl** [comment]
Thanks! That seems to have fixed the disappearing bash issue. 

However, I'm still unable to cleanly uninstall msys2 since it leaves entries in apps & features.

**Comment by StarWolf3000** [comment]
Open the Registry Editor (`regedit`), navigate to `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Uninstall` and there should be a few entries, that look like this: `{44fce769-6d66-43ce-81a9-61d9d0e86eb4}`
Check them out which is your MSYS2 installation, since I don't have this version installed on my machine here.
After you have uninstalled or deleted this version from your Windows, delete the corresponding key (the "folder") under this tree. This removes the ghost entry from Apps & Features.

**Comment by jocelynjyl** [comment]
I double checked my regedit and still can't find any registry keys associated with MSYS2. However the ghost entries remain. Is there anywhere it could be hiding other than in `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Uninstall` ?

**Comment by xbelanch** [comment]
Maybe it's no the right thread but I got two ghost entries at Apps & features of previous installations of MSYS2:

![Captura](https://user-images.githubusercontent.com/23429/96549103-50da6980-12af-11eb-91e0-d55ad484e12e.PNG)

I searched the registry keys of MSYS2 of those ghost entries on ``HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Uninstall `` but I didn't find them (as @jocelynjyl says) 

Is there any other workaround to remove those ghost entries? I found this on Stack Overflow https://stackoverflow.com/questions/50545907/how-to-uninstall-msys2 but I think that's not the properly way to solve that.

**Comment by StarWolf3000** [comment]
@xbelanch If the ghost entries no longer exist on the installed location, clicking on "Uninstall" (or "Desinstalar" for your UI language) will Windows make tell you that the setup program or uninstall program cannot be found and asking you to remove the entry from list.

Be careful! If you installed the new version into the same location, clicking "Uninstall" will remove it instead, even when the older entry is selected!

**Comment by xbelanch** [comment]
>Be careful! If you installed the new version into the same location, clicking "Uninstall" will remove it instead, even when the older entry is selected!

Yep! I know! (appreciate your warning) That's why I'm looking for a solution to remove those old MSYS2 entries from Apps & features.
